app/time_tracking/work_time/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.utils.timesince import timesince
+import time
+
+class WorkTime(models.Model):
+    start_time = models.DateTimeField(blank=False, null=False)
+    end_time   = models.DateTimeField(blank=True, null=True)
```

Comment:
Why is this nullable

Reply:
This is to disengage between having:
* an active working session, on this case the `end_time` is null
* and an ended session, in this case, the `end_time` is set to the end datetime value

Reply:
I removed the redundant `blank=False, null=False`